webcrypto/aes.go

```diff
@@ -75,7 +75,7 @@ func (akgp *AesKeyGenParams) GenerateKey(
 		return nil, NewError(0, OperationError, "invalid key length")
 	}
 
-	randomKey := make([]byte, akgp.Length/8)
+	randomKey := make([]byte, bitLength(akgp.Length).asByteLength())
```

Comment:
Have you considered making `AesKeyGenParams.Length` of type `bitLength` to avoid the conversion?

Reply:
I was a bit on the fence about it; but as you bring it up, I'll give it a shot, let's see if Goja handles that well 🤞🏻

Reply:
Goja was happy 😀 Applied the change indeed, thanks a lot 👍🏻 🙇‍♂️